census_data_downloader/language.py

```diff
@@ -10,6 +10,7 @@
 @register_downloader
 class HouseholdLanguageDownloader(BaseDownloader):
     YEAR_LIST = (2017, 2016)
+    GEO_LIST = ("nationwide", "state", "county", "congressional_district", "place")
```

Comment:
I think the class that needs this now is my new Longform download I made after you probably started in on this already.